### PR TITLE
feat: Create minimal README for schema store

### PR DESCRIPTION
**Pull request by mimotej**
SSIA
part of: https://github.com/operate-first/SRE/issues/107

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,11 @@
...
+# Operate First - OpenShift JSON schemas
 
-Derive new repositories from this template
+This repository stores OpenShift JSON schemas used with Kubeval tool to validate YAML Kubernetes configuration files.
 
-List of featurese:
+## Structure of the repository
 
-## License
+JSON files are stored in `schemas/master-standalone-strict`.  Master name is used in Kubeval to define default version of Kubernetes and standalone is used for schemas, that are de-referenced (meaning without URL references) and can be used as standalone documents. Strict is used for schemas which do no allow arbitrary attributes.
```
Comment (HumairAK) [inline README.md]: Find this section a bit confusing, seems like you want to explain the reasoning behind the way you named the directory. I would instead reword it like this:

---
 Schemas are stored as JSON files in `schemas/` at the root of the repo. The directories within `schemas` are named using the following format: `${k8s-version}-${mode}-${strictvalue}`, where: 
```
k8s-version = bla bla 
mode = bla bla 
strictvalue = bla bla  
```

I also don't fully understand what you are trying to say here:

> that are de-referenced (meaning without URL references)

In other words not sure what standalone means.

**Comment by HumairAK** [comment]
documentation seems to lack explicit steps on how to recreate/update, I would include actual commands in the doc or at least a script of some sort to update the schemas, or generate a new set of schemas.

File: README.md (hunk 1 above)
Comment (mimotej) [reply README.md]: Thank you for an idea :+1: . I rewrote the README so it is hopefully little bit more helpful.

**Review by tumido — APPROVED** [review]
/lgtm
/approve
